apps/emacs/emacs.py

```diff
@@ -0,0 +1,165 @@
+from talon import Context, Module, actions
+import logging
+
+mod = Module()
+setting_meta = mod.setting(
+    'emacs_meta',
+    type=str,
+    default='esc',
+    desc="""What to use for the meta key in emacs. Defaults to 'esc', since that should work everywhere. Other options are 'alt' and 'cmd'.""",
```

Comment:
Does this mean you would have to change your emacs configuration to use escape or your talon configuration to use another choice?